master/usage/configuration/ip-in-ip.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Configuring IP-in-IP
+---
+
+If your network fabric performs source/destination address checks 
+and drops traffic when those addresses are not recognized, it may be necessary to
+use IP-in-IP encapsulation of the inter-workload traffic.
+  
+This is often the case for public-cloud environments where you have limited control
+over the fabric and you are not peering your Calico BGP network with the rest of the
+network.
+
+Calico can be configured to use IP-in-IP encapsulation by enabling the IPIP option
+on the [IP pool resource]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool). 
+When enabled, Calico will use IP-in-IP encapsulation when routing packets *to*
+workload IPs falling in the IP pool range.
+
+An optional `mode` field toggles when IP-in-IP is used:
+-  A value of `always` is used when IP-in-IP is always required.  This is the default
+   value used if the `mode` field is omitted in the configuration.
+-  A value of `cross-subnet` is used when IP-in-IP is only required when routing 
+   between hosts on different subnets.
+   
+> **NOTE**
+>
+> Routing between Workloads and Host Endpoints running Calico is asymmetric when
+> IPIP is enabled, this causes traffic to be discarded between them.  To resolve 
+> this, enable nat-outgoing or use the IPIP tunnel address on the host
+> for the Host Endpoint IP address.
+
+### Configuring cross-subnet IP-in-IP
+
+The following `calicoctl` example will create or modify an IPv4 pool with 
+CIDR 192.168.0.0/16 using IPIP mode `cross-subnet`.
+
+```
+$ calicoctl apply -f - << EOF
+apiVersion: v1
+kind: ipPool
+metadata:
+  cidr: 192.168.0.0/16
+spec:
+  ipip:
+    enabled: true
+    mode: cross-subnet
+EOF
+```
+
+> **Note** 
+>
+> The `cross-subnet` mode option requires each Calico node to be configured
+> with the IP address and subnet of the host.  However, the subnet configuration
+> was only introduced in Calico v2.1.  If any nodes in your deployment were originally
+> created with an older version of Calico, or if you if you are unsure whether 
+> your deployment is configured correctly, follow the steps in 
+> [Upgrading from pre-v2.1](#upgrading-from-pre-v21) before enabling "cross-subnet" IPIP.
+
+#### Upgrading from pre-v2.1
```

Comment:
Suggestion, order the doc as follows

1. What's IPIP / Why do you Need it
2. Enabling IPIP (with default `mode: always`)
3. Enabling IPIP cross-subnet only
    -  Why
    - Requirements
        - Calico v2.1
        - Nodes configured with subnet
            - How to check you meet this requirement
            - Warning that upgraded nodes won't have it
            - link to how to fix it if it's not
    - Enabling It